pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -231,8 +235,9 @@ func newCertRotationController(
 			Name:      "kube-apiserver-to-kubelet-client-ca",
 			AdditionalAnnotations: certrotation.AdditionalAnnotations{
 				JiraComponent:                    "kube-apiserver",
-				AutoRegenerateAfterOfflineExpiry: "https://github.com/openshift/cluster-kube-apiserver-operator/pull/1631,'[sig-cli] Kubectl logs logs should be able to retrieve and filter logs  [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]'",
 				Description:                      "CA for the kubelet to recognize the kube-apiserver client certificate.",
+				TestName:                         "[sig-cli] Kubectl logs logs should be able to retrieve and filter logs  [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]",
```

Comment:
coderabbitai points out duplicate `logs` in test name.

Reply:
Yes, this is actual test name though - its comprised from suite ["Kubectl logs"](https://github.com/openshift/kubernetes/blob/a237dcbeee8bbe1440c94349399cc6eac8870910/test/e2e/kubectl/logs.go#L125), test group ["logs"](https://github.com/openshift/kubernetes/blob/a237dcbeee8bbe1440c94349399cc6eac8870910/test/e2e/kubectl/logs.go#L143) and test name [" should be able to retrieve"](https://github.com/openshift/kubernetes/blob/a237dcbeee8bbe1440c94349399cc6eac8870910/test/e2e/kubectl/logs.go#L167)